packages/next-codemod/bin/upgrade.ts

```diff
@@ -263,3 +270,61 @@ async function suggestCodemods(
 
   return codemods
 }
+
+async function suggestReactCodemods(packageManager: PackageManager) {
+  const { runReactCodemod } = await prompts(
+    {
+      type: 'toggle',
+      name: 'runReactCodemod',
+      message: 'Do you want to run the React codemod?',
+      initial: true,
+      active: 'Yes',
+      inactive: 'No',
+    },
+    { onCancel }
+  )
+
+  if (runReactCodemod) {
+    const commandMap = {
+      yarn: 'yarn dlx',
+      pnpm: 'pnpx',
+      bun: 'bunx',
+      npm: 'npx',
+    }
+    const command = commandMap[packageManager] || 'npx'
+
+    // https://react.dev/blog/2024/04/25/react-19-upgrade-guide#run-all-react-19-codemods
+    execSync(`${command} codemod@latest react/19/migration-recipe`, {
+      stdio: 'inherit',
+    })
+  }
+}
+
+async function suggestReactTypesCodemods(packageManager: PackageManager) {
+  const { runReactTypesCodemod } = await prompts(
+    {
+      type: 'toggle',
+      name: 'runReactTypesCodemod',
+      message: 'Do you want to run the React Types codemod?',
+      initial: true,
+      active: 'Yes',
+      inactive: 'No',
+    },
+    { onCancel }
+  )
+
+  if (runReactTypesCodemod) {
+    const commandMap = {
+      yarn: 'yarn dlx',
+      pnpm: 'pnpx',
+      bun: 'bunx',
+      npm: 'npx',
+    }
+    const command = commandMap[packageManager] || 'npx'
+
+    // https://react.dev/blog/2024/04/25/react-19-upgrade-guide#typescript-changes
+    execSync(`${command} types-react-codemod@latest preset-19 .`, {
+      stdio: 'inherit',
+    })
```

Comment:
Have you tested that this works properly with the prompt you get from `preset-19`?

Reply:
Yes, LGTM

![CleanShot 2024-10-11 at 00 10 27](https://github.com/user-attachments/assets/d5c837a3-8ab1-47b3-82f4-85a59f7fdb02)